es-mx: Using v1.2.1 of covid-19.po and v1.0.1 of ebola.po file

Delizald opened the pull request.

Delizald committed:
Using v1.2.1 of covid-19.po and v1.0.1 of ebola.po file

tematim commented on Localization/es-MX/Diseases/kidney-cancer.po:
```diff
@@ -0,0 +1,31 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

Delizald replied:
This is completed.

tematim commented on Localization/es-MX/Diseases/parkinson's-disease.po:
```diff
@@ -0,0 +1,27 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

Delizald replied:
This is completed.

tematim commented on Localization/es-MX/Diseases/epigentics.po:
```diff
@@ -0,0 +1,20 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

Delizald replied:
This is completed.

tematim commented on Localization/es-MX/Diseases/ebola-virus.po:
```diff
@@ -0,0 +1,32 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

Delizald replied:
This is completed.

tematim commented on Localization/es-MX/Diseases/dengue-fever.po:
```diff
@@ -0,0 +1,29 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

Delizald replied:
This is completed.

tematim commented on Localization/es-MX/Diseases/P53.po:
```diff
@@ -0,0 +1,27 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

Delizald replied:
This is completed.

tematim commented on Localization/es-MX/Diseases/chagas-disease.po:
```diff
@@ -0,0 +1,27 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

tematim commented on Localization/es-MX/Diseases/hepatitis-c.po:
```diff
@@ -0,0 +1,29 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

Delizald replied:
This is completed.

tematim commented on Localization/es-MX/Diseases/huntington's-disease.po:
```diff
@@ -0,0 +1,27 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

Delizald replied:
This is completed.

tematim commented on Localization/es-MX/Diseases/zika-virus.po:
```diff
@@ -0,0 +1,29 @@
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Remove this line to allow the merge or add double quotes before the word Language

Delizald replied:
This is completed.

Delizald committed:
adding missing quotes to languague files as per tematim's review

Delizald committed:
fixed non escaped word in quotes in huntington's disease.po

meatball133 committed:
Update ebola-virus.po

Really good job. I see that the source version and tagret version isn't the same here tho

meatball133 commented:
Looks good for me, really good job!

meatball133 committed:
Update covid-19.po
